src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -278,6 +273,10 @@ object SnowflakeTelemetry {
 
       // Add Spark configuration
       val sparkConf = SparkEnv.get.conf
+      metric.put(TelemetryClientInfoFields.SPARK_APPLICATION_ID,
```

Comment:
1. Add `spark_application_id` for spark application ID
2. Add `is_pyspark` for whether it is submitted by pyspark.